src/pages/setpassword/SetPasswordPageProps.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+
+export default {
+    propTypes: {
+        /* Onyx Props */
+
+        // The details about the account that the user is signing in with
+        account: PropTypes.shape({
+            // An error message to display to the user
+            error: PropTypes.string,
+
+            // Whether or not a sign on form is loading (being submitted)
+            loading: PropTypes.bool,
+        }),
+
+        // The credentials of the logged in person
+        credentials: PropTypes.shape({
+            // The email the user logged in with
+            login: PropTypes.string,
+
+            // The password used to log in the user
+            password: PropTypes.string,
+        }),
```

Comment:
I see all the above props being passed as `propTypes` to `SetPasswordPage`, but that's not the component that is receiving these props.

Reply:
In fact, it looks like nothing is connected to Onyx so none of these props will be available.